manifests/base/cuda-rstudio-buildconfig.yaml

```
@@ -0,0 +1,144 @@
+---
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  labels:
+    app: buildchain
+    component: cuda-image
+  name: cuda-rhel9
+spec:
+  name: latest
+  lookupPolicy:
+    local: true
+---
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com/red-hat-data-services/notebooks/tree/main/rstudio"
+    opendatahub.io/notebook-image-name: "CUDA - R Studio"
+    opendatahub.io/notebook-image-desc: "R Studio workbench image, allows to integrated development environment for R, a programming language for statistical computing and graphics."
+    opendatahub.io/recommended-accelerators: '["nvidia.com/gpu"]'
+  name: cuda-rstudio-rhel9
+spec:
+  name: latest
+  lookupPolicy:
+    local: true
+---
+kind: BuildConfig
+apiVersion: build.openshift.io/v1
+metadata:
+  name: cuda-rhel9
+  labels:
+    app: buildchain
+    component: cuda-rhel9
+spec:
+  source:
+    type: Git
+    git:
+      uri: "https://github.com/red-hat-data-services/notebooks"
+      ref: f/rstudio-rhel9
```

```suggestion
      ref: main
```

lets move it to main, as we have merged the content to main.